src/lib/multipageloader.cc

```diff
@@ -435,16 +435,15 @@ void ResourceObject::amfinished(QNetworkReply * reply) {
 		QString extension = fi.completeSuffix().toLower().remove(QRegExp("\\?.*$"));
 		bool mediaFile = settings::LoadPage::mediaFilesExtensions.contains(extension);
 		if ( ! mediaFile) {
-			// XXX: Notify network errors as higher priority than HTTP errors.
-			//      QT's QNetworkReply::NetworkError enum uses values overlapping
-			//      HTTP status codes, so adding 1000 to QT's codes will avoid
-			//      confusion. Also a network error at this point will probably mean
-			//      no HTTP access at all, so we want network errors to be reported
-			//      with a higher priority than HTTP ones.
-			//      See: http://doc-snapshot.qt-project.org/4.8/qnetworkreply.html#NetworkError-enum
-			error(QString("Failed to load %1, with network status code %2 and http status code %3 - %4")
-				.arg(reply->url().toString()).arg(networkStatus).arg(httpStatus).arg(reply->errorString()));
-			httpErrorCode = networkStatus > 0 ? (networkStatus + 1000) : httpStatus;
```

Comment:
Why did you remove this?

Reply:
I couldn't find where the `httpErrorCode` variable was used later / I think it's unused.

Reply:
If the error message includes both the `networkStatus` and `httpStatus` variables, I'm not sure this logic is really necessary any more.
If you consider this point valid I can also amend lines 452-459 in the same way (show `networkStatus` and `httpStatus` in the error / warning message)

Reply:
@ashkulz thoughts on this?

Reply:
@ashkulz ping

Reply:
re-pinging @ashkulz

Reply:
pong